USAGE.md

```diff
@@ -36,16 +37,58 @@ You'll need to start `bg-carbon-cache` instead of `carbon-cache` to run carbon w
 biggraphite. If carbon is installed in `/opt/graphite/lib` and not in site-packages
 you'll need to set do set `PYTHONPATH=/opt/graphite/lib`.
 
+To use Elasticsearch as a metadata backend, replace `BG_DRIVER` parameter by
+`BG_DATA_DRIVER` and `BG_METADATA_DRIVER` parameters and provide informations
+to connect to Elasticsearch:
+
+```text
+# Usual configuration
+...
+
+# Cassandra configuration
+BG_DATA_DRIVER = cassandra
+BG_CASSANDRA_KEYSPACE = biggraphite
+BG_CASSANDRA_CONTACT_POINTS = 127.0.0.1
+
+## Elasticsearch configuration
+BG_METADATA_DRIVER = elasticsearch
+BG_ELASTICSEARCH_HOSTS = 127.0.0.1
+BG_ELASTICSEARCH_PORT = 9200
+```
+
+If Elasticsearch needs credentials, they can be provided by
+`BG_ELASTICSEARCH_USERNAME` and `BG_ELASTICSEARCH_PASSWORD` either in the
+configuration file either as environment variables.
+
 ## Graphite Web plugin
 
 In local_settings.py set the following settings:
 
 ```python
 STORAGE_FINDERS = ['biggraphite.plugins.graphite.Finder']
 TAGDB = 'biggraphite.plugins.tags.BigGraphiteTagDB'
+BG_DRIVER = 'cassandra'
 BG_CASSANDRA_KEYSPACE = 'biggraphite'
 BG_CASSANDRA_CONTACT_POINTS = '127.0.0.1'
 ```
 
 You can optionally keep the StandardFinder to enable double reads from whisper
-and biggraphite.
\ No newline at end of file
+and biggraphite.
+
+To use Elasticsearch as a metadata backend, replace `BG_DRIVER` parameter by
```

Comment:
The following seems copy/pasted. Can we add a link instead ?

Reply:
I'll simplify this, then